sktree/tree/_unsup_splitter.pyx

```diff
@@ -216,9 +147,11 @@ cdef class BestSplitter(BaseDenseSplitter):
         cdef DTYPE_t[::1] Xf = self.feature_values
```

Comment:
Xf @ unsup_splitter

Reply:
?

Reply:
Just a reference point to remind myself where Xf is being used